optimizely/config/polling_manager.go

```diff
@@ -27,16 +27,26 @@ import (
 	"github.com/optimizely/go-sdk/optimizely/logging"
 )
 
-const defaultPollingWait = time.Duration(5 * time.Minute) // default 5 minutes for polling wait
+const defaultPollingInterval = time.Duration(5 * time.Minute) // default to 5 minutes for polling
+
+// DatafileURLTemplate is used to construct the endpoint for retrieving the datafile from the CDN
+const DatafileURLTemplate = "https://cdn.optimizely.com/datafiles/%s.json"
 
 var cmLogger = logging.GetLogger("PollingConfigManager")
 
+// PollingProjectConfigManagerOptions used to create an instance with custom configuration
+type PollingProjectConfigManagerOptions struct {
+	FallbackDatafile []byte
```

Comment:
Why not just call this `Datafile`? It is technically not the fallback datafile but the initial datafile so if not `Datafile` may be `InitialDatafile` will be more accurate than `FallbackDatafile` because that suggests that if network is not reachable then that is the datafile it will fall back to which will not be the case it will be datafile that was most recently polled. Does that make sense?

Reply:
Yeah I'll make that change